syntax = "proto3";
package cs;

enum ProtoID {
    ID_NULL = 0;
    ID_C2S_Ping = 10001;
    ID_S2C_Ping = 10002;
    ID_C2S_Login = 10003;
    ID_S2C_Login = 10004;
}

message C2S_Ping {
    uint32 send_time = 1;
}

message S2C_Ping {
    uint32 send_time = 1;
    uint32 recv_time = 2;
}

message C2S_Login {
    uint64 uid = 1; 
}

message S2C_Login {
    uint32 ret = 1;
}